notebooks/api/0.9/07-custom-policy.ipynb

```diff
@@ -22,7 +22,7 @@
    },
    "outputs": [],
    "source": [
-    "node = sy.orchestra.launch(name=\"test-domain-1\", port=8080, dev_mode=True)"
+    "node = sy.orchestra.launch(name=\"test-domain-1\", port=8080, dev_mode=True, reset=True)"
```

Comment:
Q:  I am unable to reproduce the error mentioned in the ticket
https://github.com/OpenMined/Heartbeat/issues/236

by setting reset=True and running the notebooks tests in `dev` branch of custom policies

Does the ticket aim to add serde for Custom Policies?

Reply:
@rasswanth-s it is related to both notebook tests and custom policy serde.

The issue can be replicated when we run the notebook `custom-policies.ipynb` notebook (skip node.land()) and post that in a new network or ipython shell run the following code or you can refer to the [notebooks in this commit](https://github.com/OpenMined/PySyft/tree/cb6827811dfbcdf7dfaa06a3de880fa078ad3197/notebooks/khoa/custom_policy_serde_bug)

```
import syft as sy
from syft.client.api import SyftAPICall

my_domain_client = sy.login(email="info@openmined.org", password="changethis", port=8080)
api_call = SyftAPICall(
    node_uid=my_domain_client.id, path="request.get_all", args=(), kwargs={}, blocking=True
)

signed_api_call = api_call.sign(my_domain_client.credentials)

response = my_domain_client.connection.make_call(signed_call=signed_api_call)

response.message
```